src/wave.js

```diff
@@ -195,11 +206,14 @@ export class Wave extends mix(WaveBase).with(
         while (this.structureGroup.children.length) {
             this.structureGroup.remove(this.structureGroup.children[0]);
         }
+
+        while (this.labelRenderer.domElement.firstChild) this.labelRenderer.domElement.removeChild(this.labelRenderer.domElement.firstChild);
```

Comment:
If we already added labels to structureGroup in https://github.com/Exabyte-io/wave.js/pull/40/files#diff-7cb304ed282e9e66b1e3dd826ffa0c6aR116 above, why do we need to clear it separately??